crypto/store/store_local.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
`_locl.h` is the most common, and also the most popular choice for new files, it seems...  so yeahok.